apps/storybook.namekit.io/stories/Namekit/TruncatedText.stories.tsx

```diff
@@ -1,44 +1,36 @@
 import { TruncatedText } from "@namehash/namekit-react/client";
 import type { Meta, StoryObj } from "@storybook/react";
 
-enum TruncatedTextExample {
-  SHORT_TEXT = "SHORT_TEXT",
-  LONG_TEXT = "LONG_TEXT",
-}
-
-const getExampleTruncatedText = (example: TruncatedTextExample): string => {
-  switch (example) {
-    case TruncatedTextExample.SHORT_TEXT:
-      return "heyiamsmall";
-    case TruncatedTextExample.LONG_TEXT:
-      return "heyiamaverylongtextthatkeepsgrowingunstopablyomgiamstillgoingonokdone";
-  }
-};
+const SHORT_TEXT = "heyiamsmall";
+const LONG_TEXT = "heyiamaverylongtextthatkeepsgrowingunstopablyomgiamstillgoingonokdone";
 
 const meta: Meta<typeof TruncatedText> = {
   component: TruncatedText,
   title: "Namekit/TruncatedText",
   argTypes: {
```

Comment:
I learned that storybook doesn't currently give a way to sort the display of these controls in their UI. That's a bummer. Related issue here: https://github.com/storybookjs/storybook/discussions/24524

But ultimately this isn't much of an issue because there's a better way to document our Storybook stories. I'll show this in a separate PR later.